README.md

```diff
@@ -0,0 +1,66 @@
+The canonical URL for the upstream Spectrum git repository is
+<https://spectrum-os.org/git/spectrum/>.
+
+To try Spectrum OS, you need to build it from source and then install it to an SD card.
+
+> Cross-compiled version of Spectrum OS is for i.MX 8QXP board only.
```

Comment:
As of now, cross-compiled version of ...

Reply:
Changed to: 
Currently, the cross-compiled version of Spectrum OS is for i.MX 8QXP board only.